src/h3lib/include/polygon.h

```diff
@@ -17,20 +17,15 @@
  * @brief Polygon algorithms
  */
 
-#ifndef POLYGON_H
-#define POLYGON_H
+#ifndef POLYGON_GEOFENCE_H
```

Comment:
Why rename if we're not renaming the file?

Reply:
That change should have been backed out